packages/react/types/index.d.ts

```diff
@@ -94,8 +92,42 @@ export interface ClassNamesProps {
  */
 export function ClassNames(props: ClassNamesProps): ReactElement
 
-declare module 'react' {
-  interface Attributes {
-    css?: Interpolation<Theme>
+type WithConditionalCssProp<P> = P extends { className?: string }
```

Comment:
I think this should be:
```tsx
type WithConditionalCssProp<P> = "className" extends keyof P
  ? P extends { className?: string }
    ? P & { css?: Interpolation<Theme> }
    : P
  : P;
```

So that it doesn't add className for `{}`

Reply:
Eh, I always forget about this surprising, to me, behavior of the `{}`. Good catch.